src/azure-cli/azure/cli/command_modules/lab/validators.py

```diff
@@ -530,60 +518,3 @@ def _single(collection):
 
 def _any(collection):
     return len([x for x in collection if x]) > 0
-
-
-def _get_current_user_object_id(graph_client):
-    from msrestazure.azure_exceptions import CloudError
-    try:
-        current_user = graph_client.signed_in_user.get()
-        if current_user and current_user.object_id:  # pylint:disable=no-member
-            return current_user.object_id  # pylint:disable=no-member
-    except CloudError:
-        pass
-    return None
-
-
-def _get_object_id(graph_client, subscription=None, spn=None, upn=None):
-    if spn:
-        return _get_object_id_by_spn(graph_client, spn)
-    if upn:
-        return _get_object_id_by_upn(graph_client, upn)
-    return _get_object_id_from_subscription(graph_client, subscription)
-
-
-def _get_object_id_from_subscription(graph_client, subscription):
-    if subscription['user']:
```

Comment:
It is not possible for a subscription to have no `user` field. It will always be set during creation:

https://github.com/Azure/azure-cli/blob/b00483c1a1cd17053af5c483f62b11c829f4c27d/src/azure-cli-core/azure/cli/core/_profile.py#L408-L415